### PR TITLE
Add a working option to english story

### PR DESCRIPTION
**Pull request by Dev-C94**
Please review this, @casey-collab

### CODE REVIEW

File: english/working/working.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+You know you should be doing it.
```
Comment (casey-collab) [inline english/working/working.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Dev-C94** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smile:

**Pull request merged**